PF/Assignment1/SortedLinkedList.java

```diff
@@ -0,0 +1,46 @@
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+
+//to implement sorted link list
+public class SortedLinkedList {
+
+	List<Integer> node = new LinkedList<Integer>();
+
+	// to add element
+	void addElement(int element) {
+		node.add(element);
+		Collections.sort(node);
+	}
+
+	// to add element in sorted list
+	public void addElements(int index, int element) {
```

Comment:
ACTION: Use proper name for method to add a element into a sorted list. Current method name shows there is a list of numbers to add.